charts/gateway/Chart.yaml

```diff
@@ -2,15 +2,15 @@ apiVersion: v2
 name: trino-gateway
 description: A Helm chart for Trino Gateway
 type: application
-version: "0.12.0"
-appVersion: "12"
+version: "1.13.0"
```

Comment:
Why do we increment a major version?

Reply:
Having 0 as the major version might be understood the chart is unstable, or not ready for production use, which is not accurate, and might confuse some users. We agreed with @mosabua to bump major versions for both Trino and Trino Gateway to 1. So there are no breaking changes here, we just want to mark a stable release from this point forward.

Reply:
Our chart used to have full major versions so .. and when we changed to the chart repo here it was changed to 0.x .. however we (Trino Gateway maintainers and @nineinchnick feel that this is misleading. The chart is totally usable for production so the version should be > 1.0 .. hence we are switching 1.x with x being the Trino Gateway version now .. and if we cut chart releases in between we go minor update.. 

We also plan to upgrade the trino chart to be >1 .. and the go client ..